tests/lib/managers/metadata-test.js

```diff
@@ -214,6 +214,52 @@ describe('Metadata', function() {
 		});
 	});
 
+	describe('deleteTemplate()', function() {
+		var scope,
+			template;
+
+		beforeEach(function() {
+
+			scope = 'enterprise';
+			template = 'testtemplate';
```

Comment:
If these are constant, it's okay to just declare them statically on lines 217 and 218 above, e.g.
```js
var scope = 'enterprise',
	template = 'testtemplate';
```